.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,53 @@
+name: Benchmarks Test
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  Benchmarks:
+    runs-on: ${{ matrix.runner }}
+    strategy:
+      matrix:
+        runner: [ubuntu-latest, self-hosted]
+        benchmark: [sysbench, getpid]
+      fail-fast: false
+    timeout-minutes: 60
+    container: 
+      image: asterinas/asterinas:0.5.0
+      options: --device=/dev/kvm
+    env:
+      RUSTUP_DIST_SERVER: https://mirrors.ustc.edu.cn/rust-static
+      RUSTUP_UPDATE_ROOT: https://mirrors.ustc.edu.cn/rust-static/rustup
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up the environment
+      run: |
+        sudo apt-get update && sudo apt-get install -y jq
+        chmod +x regression/benchmark/benchmarks.sh
+        git config --global --add safe.directory /__w/asterinas/asterinas
+        git config --global http.sslVerify false
+        git config --global http.version HTTP/1.1
+
+    - name: Run benchmark
+      uses: nick-invision/retry@v2
+      with:
+        timeout_minutes: 20
+        max_attempts: 3
+        command: |
+          make install_osdk
+          bash regression/benchmark/benchmarks.sh ${{ matrix.benchmark }} ${{ contains(matrix.runner, 'self-hosted') && 1 || 0 }}
+        on_retry_command: make clean
+
+    - name: Store benchmark results
+      uses: grief8/github-action-benchmark@v1
```

Comment:
Is it possible to move this action to Asterinas org? And this action---its name and arguments--does not seem to be very self-describing. For example, the name is "github-action-benchmark", but apparently, the job is not responsible for running the benchmark, only collecting and visualizing the results. So it should use a more accurate name.